konami/identifiers.md

```diff
@@ -140,5 +141,6 @@ The following variables are used across the different identifiers and can denote
 ### Soft ID code
 
 * Source: [Security code](#security-code) and [release code](#release-code) combined and shown in-game
+  * Some games change the soft ID, for example jubeat: `GQL44JBA` (security code) -> `L44:J:F:A` (in-game)
 * Format: `$g:$d:$s:$r:$e`
-* Example: `PIX:J:B:A:201807300200`
+* Example: `PIX:J:B:A:2018073002`
```

Comment:
Ditto, also for the specific example: https://twitter.com/MonochromeRay/status/1024361931635937281

Reply:
Look closer at the photo, it's ``2018073002``.
year - month - day - build

Reply:
Yeah, I'm stupid. I blame 4AM.